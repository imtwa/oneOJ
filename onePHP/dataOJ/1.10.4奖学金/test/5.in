13
93 79 91
92 88 85
99 89 90
71 94 81
79 91 71
71 87 75
74 96 81
91 89 75
92 76 80
86 75 97
94 98 88
81 86 100
97 83 80
