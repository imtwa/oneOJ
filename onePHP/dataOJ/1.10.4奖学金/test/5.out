11 280
3 278
12 267
2 265
1 263
